tests/foreman/cli/test_location.py

```diff
@@ -45,8 +35,10 @@ def test_create_location_with_different_names_positive(self, name):
         @Assert: Location is created successfully and has proper name
 
         """
-        loc = make_location({'name': name})
-        self.assertEqual(loc['name'], name)
+        for name in valid_data_list():
```

Comment:
Great, Thank you.  I had no idea of this since it is not documented in the tests.  I will create a new data creation helper for org/loc tests then.